CONTRIBUTING.rst

```diff
@@ -0,0 +1,88 @@
+Contributing
+============
+
+RGBDS was created in the late 90's and has received contributions of several
+developers since then. It wouldn't have been possible to get to this point
+without their work and. For that reason, it is always open to the contributions
+of other people.
+
+Reporting Bugs
+--------------
+
+Bug reports are essential to improve RGBDS and they are always welcome. If you
+want to report a bug:
+
+1. Make sure that there isn't a similar issue already reported
+   `here <https://github.com/rednex/rgbds/issues>`__.
+
+2. Figure out a way of reproducing it reliably.
+
+3. If there is a piece of code that triggers the bug, try to reduce it to the
+   smallest file you can.
+
+4. Create a new `issue <https://github.com/rednex/rgbds/issues>`__.
+
+Of course, it may not always be possible to give an accurate bug report, but it
+always helps to fix it.
+
+Requesting new features
+-----------------------
+
+If you think you have a good idea that should be implemented, you can propose it
+to be done.
+
+1. Create a new `issue <https://github.com/rednex/rgbds/issues>`__.
+
+2. Try to be as accurate as possible. Describe what you need and why you need
+   it, maybe with examples.
+
+You need to understand that the contributors are doing it in their free time, so
```

Comment:
This feels a little forceful to me, how about “Please understand that...” or “Please remember that...”?

Reply:
Done